components/frontend/src/header_footer/Footer.test.jsx

```diff
@@ -18,7 +18,7 @@ it("renders a quote when there is no report", async () => {
 
 it("renders a link to the report url", async () => {
     const { container } = render(<Footer report={{ title: "Report title" }} />)
-    expect(screen.getByText("Report title").closest("a")).toHaveAttribute("href", "http://localhost/")
+    expect(screen.getByText("Report title").closest("a")).toHaveAttribute("href", "http://localhost:3000/")
```

Comment:
Why is the port now needed?

Reply:
I'm not sure exactly, but I think it's because vitest reads the vite config file, so it knows the URL that serves the frontend in dev mode.